src/components/chat/chat.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import ForumIcon from '@material-ui/icons/Forum';
+import CancelIcon from '@material-ui/icons/Cancel';
+import MailOutlineIcon from '@material-ui/icons/MailOutline';
+import MessengerCustomerChat from 'react-messenger-customer-chat';
+import { useSelector } from 'react-redux';
+import { Button, TextField } from '@material-ui/core';
+import { useStyles } from './chat.style';
+import { CHAT } from '../../translations/chat.translation';
+import kyivstar from '../../images/kyivstar.png';
+import vodafone from '../../images/vodafone.png';
+
+export const Chat = () => {
+  const [visible, setBlock] = useState(false);
+  const [mailFormVisible, setMailFormVisible] = useState(false);
+  const { language, themeMode } = useSelector((state) => ({
+    language: state.Language.language,
+    themeMode: state.Theme.lightMode
+  }));
+  const style = useStyles({ themeMode, visible, mailFormVisible });
+
+  const activeMessenger = () => (
+    <div className={style.formFieldActive}>
+      <span className={style.mailTitle}>{CHAT[language].sendMail}.</span>
+      <TextField
+        className={style.dataInput}
+        id='outlined-basic'
+        label={CHAT[language].name}
+        variant='outlined'
+        size='small'
+        multiline
+        rows={1}
+      />
+      <TextField
+        className={style.dataInput}
+        id='outlined-basic'
+        label={CHAT[language].email}
+        variant='outlined'
+        size='small'
+        multiline
+        rows={1}
+      />
+      <TextField
+        className={style.dataInput}
+        id='outlined-basic'
+        label={CHAT[language].msgText}
+        variant='outlined'
+        size='small'
+        multiline
+        rows={4}
+      />
+      <Button className={style.btnSend}>{CHAT[language].sendBtn}</Button>
+    </div>
+  );
+
+  return (
+    <div>
+      <div className={visible ? style.iconsMessengers : style.disable}>
+        <div>
+          <MessengerCustomerChat
+            pageId='106973281128068'
```

Comment:
Please, make the `pageId` and `appId` environment variables  with the same approach as `REACT_APP_API_URL` is implemented

Reply:
done